statick_tool/statick.py

```diff
@@ -83,6 +83,18 @@ def set_logging_level(args: argparse.Namespace) -> None:
         logging.root.setLevel(log_level)
         logging.info("Log level set to %s", args.log_level.upper())
 
+    @classmethod
+    def set_cpu_count(cls, arg: str) -> int:
+        """Set correct number of CPU cores to use."""
+        max_cpus = multiprocessing.cpu_count()
+        desired = int(arg)
```

Comment:
Nit, but `arg` is a vague argument name.  Can this be called something like `num_cpus`?

Reply:
[4514d04](https://github.com/sscpac/statick/pull/460/commits/4514d0465224ad043ccff760ffcdd769eb818388)